iothub/service/src/DigitalTwin/Models/UpdateDigitalTwinOptions.cs

```diff
@@ -15,13 +17,12 @@ public class UpdateDigitalTwinOptions
         /// The request's operation is performed only if this ETag matches the value maintained by the server,
         /// indicating that the entity has not been modified since it was last retrieved.
         /// <para>
-        /// To perform the operation only if the entity exists, set the ETag to the wildcard character <c>"*"</c>.
-        /// To perform the operation unconditionally, leave it the default value of <c>null</c>.
```

Comment:
q - are we losing this feature?
`To perform the operation only if the entity exists, set the ETag to the wildcard character <c>"*"</c>`. 
If yes, then it might be worth documenting this in the migration doc.

Reply:
Tim said it didn't make any sense. You can't patch an item that doesn't already exist.

Reply:
Was it a relic of the "CreateOrUpdate" style APIs?

Reply:
Not sure. Probably a copy pasta.